corehq/apps/domain/static/domain/js/manage_alerts.js

```diff
@@ -1,10 +1,19 @@
 hqDefine("domain/js/manage_alerts",[
     'jquery',
+    'knockout',
+    'underscore',
     'hqwebapp/js/initial_page_data',
-], function ($, initialPageData) {
+], function ($, ko, _, initialPageData) {
+
+    var domainAlert = function (options) {
+        var self = ko.mapping.fromJS(options);
+        self.editUrl = initialPageData.reverse('domain_edit_alert', self.id());
+        return self;
+    };
```

Comment:
Added this to mainly add `editUrl` to the objects.

Reply:
See [this comment](https://github.com/dimagi/commcare-hq/pull/33909/files#r1440238408)